src/commands/logistics/netid.ts

```diff
@@ -0,0 +1,115 @@
+import { CommandInteraction, ReactionUserManager } from "discord.js";
+// import cryptoRandomString from 'crypto-random-string';
+import User from "../../models/user";
+import Verification from "../../models/verification";
+import getEmailString from "../../verification-text";
+import { SlashCommandBuilder } from '@discordjs/builders';
+import got from "got/dist/source";
+import { VERIFY_TOKEN_EXPIRATION, VERIFY_TOKEN_LENGTH } from "../../constants";
+
+const ERROR_MSG = 'Could not verify at this time. Please try again later.'
+
+module.exports = {
+    data: new SlashCommandBuilder()
+        .setName('netid')
+        .setDescription('Does NetID Authorization for the server')
+        .addStringOption((option: any) => option.setName('netid').setDescription('String of words seperated by a space')),
+
+    async execute(interaction: CommandInteraction) {
+        let netid = interaction.options.getString('netid');
+
+        if (!netid) {
+            interaction.reply('Usage: `/netid <netid>`');
+            return;
+        }
+
+        netid = netid.trim().split(' ')[0].toLocaleLowerCase();
+
+        if (netid.length <= 1) {
+            interaction.reply('The given netID is invalid. Usage: `/netid <netid>`');
+            return;
+        }
+
+        // Check if netid is already verified
+        let user = User.findOne({ netid: netid }).catch(err => {
```

Comment:
Should we check if the user is already verified by discord username as well? Otherwise it looks like users may be able to double-authenticate (e.g. authenticate as one NetID, then another later on).

Reply:
Ah nvm I see that case is handled later in the func